src/dmtcpplugin.cpp

```diff
@@ -39,7 +39,6 @@
 #undef dmtcp_get_local_status
 #undef dmtcp_get_uniquepid_str
 #undef dmtcp_get_ckpt_filename
-#undef dmtcp_set_coord_ckpt_dir
 #undef dmtcp_get_coord_ckpt_dir
```

Comment:
If we decide to remove `dmtcp_get_coord_ckpt_dir` then let's remove this line as well.